features/users/user_delete.feature

```diff
@@ -60,6 +60,7 @@ Scenario: Delete a user with a collection
   When I am logged in as "moderator" with password "password"
     And all emails have been delivered
     And I create the collection "fake"
+    And all indexing jobs have been run
```

Comment:
This can be removed now that `I go to the collections page` runs the indexing jobs.

Reply:
@sarken I'm not sure if it's an issue with how I set it up or something else, but I get test failures when I remove that indexing line anywhere but the "collection_browse" tests. It looks like, at least in this file, the "I go to the collections page" step is triggering this more generic step: https://github.com/otwcode/otwarchive/blob/b5bf2e7a5cd362d1e350f2be087e8049721a2484/features/step_definitions/web_steps.rb#L15.
Any ideas on what to tweak to fix this?

Reply:
I think the problem is in features/support/paths.rb:

https://github.com/littlelines/otwarchive/blob/b0f34ddd59d77fcbc02ada8906e693173bddb270/features/support/paths.rb#L24-L26

It should just be `when /^the collections page$/i`, not `when /^go to the collections page$/i`.